src/rollup/RollupCreator.sol

```diff
@@ -112,9 +115,12 @@ contract RollupCreator is Ownable {
         emit RollupCreated(
```

Comment:
let's also emit `rollupEventInbox`, `challengeManager`

or maybe we just emit `(rollup, ContractDependencies, proxyAdmin)`

Reply:
Thanks Harry!
Added both to the event.